utility_analysis_new/dp_engine.py

```diff
@@ -80,8 +92,6 @@ def _check_utility_analysis_params(params: pipeline_dp.AggregateParams,
                 {pipeline_dp.Metrics.COUNT, pipeline_dp.Metrics.SUM}))
         raise NotImplementedError(
             f"unsupported metric in metrics={not_supported_metrics}")
-    if public_partitions is None:
```

Comment:
The test for this can now be removed in dp_engine_test, it will fail on expected error

Reply:
Thanks, the test is removed

Reply:
👍  ✂️